src/resource_clients/user.ts

```diff
@@ -35,7 +35,10 @@ export class UserClient extends ResourceClient {
         };
         try {
             const response = await this.httpClient.call(requestOpts);
-            return cast(parseDateFields(pluckData(response.data)));
+            return cast(parseDateFields(
+                pluckData(response.data),
+                // Convert  monthlyUsage.dailyServiceUsages[].date to Date (by default it's ignored by parseDateFields)
+                /* shouldParseField = */ (key) => key === 'date'));
```

Comment:
@B4nan FYI this is technically speaking a breaking change after #183 as it changes `string` to `Date`. But #183 hasn't been released yet, so it should be fine. We just need to wait with the client release.